plugins/inventory/proxmox.py

```diff
@@ -528,10 +534,11 @@ def _populate_pool_groups(self, added_hosts):
     def _populate(self):
 
         # create common groups
-        self.inventory.add_group(self._group('all_lxc'))
-        self.inventory.add_group(self._group('all_qemu'))
-        self.inventory.add_group(self._group('all_running'))
-        self.inventory.add_group(self._group('all_stopped'))
+        default_groups = ['lxc', 'qemu', 'running', 'stopped',
+                          'qemu_running', 'qemu_stopped', 'qemu_prelaunch', 'qemu_paused']
```

Comment:
Should these groups only be added `if self.get_option('want_facts')`?

Reply:
I would add them for consistency between runs, even with want_facts = False, the groups starting with `qemu_` will be empty but still there.

Reply:
I'm mainly asking because if they are present but not filled, there will sooner or later be bug reports that they are not filled in because folks forgot to pass `want_facts=true` :) In any case, the documentation should mentioned these groups and under which conditions they are populated.

Reply:
Good point, I'll add the info in the documentation.